### DISCUSSION TITLE
How to add new blogs?

### DISCUSSION THREAD

**Discussion by NamasteIndia** [arthelokyo/astrowind#391] [Q&A]
How to add new blogs or articles?

**Comment by AndrewB-S** [comment]
Add a new file within the content collection. 
![image](https://github.com/onwidget/astrowind/assets/105535718/0ec57e58-f3bc-4009-917d-028d171e9d85)

Each of the files under /src/content/post is a page.

**Comment by sherozsamatov** [comment]
Hi, could anyone please help with this?
I am getting an error on running `npm run dev` and `npm run build`

How can I fix this?
![Snipaste_2024-03-19_07-42-39](https://github.com/onwidget/astrowind/assets/50622092/122d07e4-01ab-45b9-8b81-b71c91c2d0c1)
